lib/components/atoms/Input/Input.js

```diff
@@ -13,32 +13,28 @@ type Props = {
   placeholder?: string,
   ariaLabel?: string,
   elementLocator?: string,
+  ref: ?HTMLInputElement,
 };
 
-const Input = ({
-  id,
-  className,
-  type,
-  name,
-  value,
-  placeholder,
-  ariaLabel,
-  elementLocator,
-  ...others
-}: Props): Node => (
-  <input
-    id={id}
-    aria-label={ariaLabel}
-    className={className}
-    name={name}
-    type={type}
-    value={value}
-    placeholder={placeholder}
-    data-locator={elementLocator || `input-${type}-${name}`}
-    {...others}
-  />
+const Input = React.forwardRef(
+  (
+    { id, className, type, name, value, placeholder, ariaLabel, elementLocator, ...others }: Props,
+    ref
+  ) => (
+    <input
+      id={id}
+      ref={ref}
+      aria-label={ariaLabel}
+      className={className}
+      name={name}
+      type={type}
+      value={value}
```

Comment:
value={value} make it conditional for File 

```javascript
value={type !== 'file' ? value : null}
```

Reply:
Implemented.